airbyte-integrations/connectors/source-paypal-transaction/acceptance-test-config.yml

```diff
@@ -1,36 +1,41 @@
-# See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
-# for more information about how to configure these tests
-connector_image: airbyte/source-paypal-transaction:dev
-tests:
-  spec:
-    - spec_path: "source_paypal_transaction/spec.json"
+acceptance_tests:
+  basic_read:
+    tests:
+    - config_path: secrets/config.json
+      empty_streams:
+      - name: balances
+        bypass_reason: "value of 'last_refresh_time' field changes during every read"
```

Comment:
I think we can skip one column (last_refresh_time) rather than the whole stream - check out `ignored_fields`

Reply:
@evantahler do we have `ignored_fields` key for basic read test?

Reply:
Hmm... it might be only on the Full Refresh - https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference#test-full-refresh-sync

Tagging @alafanechere - do you think that this is the occasion to add ignored_fields to basic read?

Reply:
@evantahler @alafanechere can you please clarify if you decided anything about this stream that is skipped?

Reply:
Well, for now, we can't use `ignored_fields` for basic read, so I think bypassing the stream is the way forward - done is better than perfect!

Reply:
Wrote https://github.com/airbytehq/airbyte/issues/21737 to track this for later